### ISSUE TITLE
Deep Learning

### ISSUE THREAD

**Issue by prizer** [prizer/NLP#1]
c3dl yobi
https://yobi.navercorp.com/C3_C3_DL

**Comment by prizer** [comment]
#### self attention
target sentence: query
input sentence: key, value (대부분 key = value)
```
In current NLP work, the key and value are usually came to the same thing, therefore key=value.
```

**Comment by prizer** [comment]
#### logit
odd: p / (1-p)
logit: odd에 log 씌운 것. log( p / (1-p) )
logit의 inverse 함수: sigmoid
sigmoid의 일반화 케이스: softmax
https://opentutorials.org/module/3653/22995
![image](https://user-images.githubusercontent.com/28520698/57272623-6d8cdc80-70cf-11e9-83c7-db79ad4f3bb2.png)
